coalib/bears/Bear.py

```diff
@@ -274,6 +281,284 @@ def log_message(self, log_message, timestamp=None, **kwargs):
+                        break
+                    except UnboundLocalError:
+                        flag = 0
+                        self.err(
+                            'A given pstats method requires an argument. '
+                            'Applying default settings')
+                        break
+                    except ValueError as err:
+                        flag = 0
+                        self.err(str(err.args[0]).format(
+                            str(err.args[1]+'()'))+'. Applying '
+                                                   'default settings')
+                        break
+        return ps.strip_dirs().sort_stats('cumtime') if flag == 0 else ps
+
+    def setup_profiler_table(self, prof,
+                             profile_bears, check_trim: bool = False):
+        fd, path = tempfile.mkstemp()
+        try:
+            with fdopen(fd, 'r+') as stream:
+                ps = self.pstats_config(prof, stream, profile_bears)
+                ps.print_stats()
+                stream.flush()
+                stream.seek(0)
+                lines = stream.readlines()
+                req_lines = []
+                table_header = []
+                for line in lines:
+                    if ('function calls' not in line and
+                        'listing order was used' not in line and
+                        'Ordered by' not in line and
+                            line.strip(' \n\t') != ''):
+                        req_lines.append(line)
+                    elif line.strip(' \n\t') != '':
+                        table_header.append(line.strip())
+                channel_values = [x for x in [y.split(' ') for y in
+                                              req_lines] if x and not x == '\n']
+
+                table_data = []
+                final_table_data = []
+                for x in channel_values:
+                    table_data.append(list(filter(lambda a: a != '', x)))
+                for x in table_data:
+                    start = ''
+                    flag = 0
+                    row = []
+                    for y in x:
+                        if y.startswith('{'):
+                            flag = 1
+                        if flag == 1:
+                            start = start + y + ' '
+                        elif y != '':
+                            t = ' '.join(y.split())
+                            row.append(t.strip())
+                    if start != '':
+                        row.append(start.strip())
+                    final_table_data.append(row)
+                colored_table_data = []
+                colors = ['red', 'white', 'blue', 'yellow', 'magenta', 'green']
+                linter_table = []
+                linter_table.append(final_table_data[0])
+                if u'linter' in str(self):
+                    for i in final_table_data:
+                        if u'linter'.title() in i[5] or str(self.__class__.__name__) in i[5]:
```

Comment:
Line is longer than allowed. (93 > 80)

*Origin: LineLengthBear, Section: `linelength`.*